### PR TITLE
chore: initialize

### PR DESCRIPTION
**Pull request by kenta-elys**
Scaffolded smart contract initial structure mainly following @politeWall ’s
- get portfolio
- get total balance

### CODE REVIEW

File: src/bin/schema.rs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+use cosmwasm_schema::write_api;
+use financial_snapshot_contract::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
```
Comment (politeWall) [inline src/bin/schema.rs:2]: there's no ExecuteMsg define in your code

Comment (kenta-elys) [reply src/bin/schema.rs:2]: Got it.

File: src/error.rs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+#[derive(Error, Debug, PartialEq)]
+pub enum ContractError {
+    #[error("{0}")]
+    StdError(#[from] StdError),
+    #[error("Payment error: {0}")]
+    Payment(#[from] PaymentError),
+    #[error("{address} : Not Found")]
+    PortfolioNotFound { address: String },
+    #[error("{sender} is not the owner of the order")]
```
Comment (politeWall) [inline src/error.rs:13]: this message have no use in this contract

File: src/bin/schema.rs (hunk 1 above)
Comment (kenta-elys) [reply src/bin/schema.rs]: Done.

File: src/tests/mock/instantiate.rs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{DepsMut, Env, MessageInfo, Response, StdResult};
+
+use crate::{bindings::query::ElysQuery, states::*, types::Order};
+
+#[cw_serde]
+pub struct InstantiateMockMsg {
+    pub epoch_cycle_interval: u128,
+    pub orders: Vec<Order>,
```
Comment (politeWall) [inline src/tests/mock/instantiate.rs:9]: there's no such thing as Order in this contract

File: src/types/portfolio/impls/new_dummy.rs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+impl Portfolio {
```
Comment (politeWall) [inline src/types/portfolio/impls/new_dummy.rs:1]: Portfolio is not define here

File: src/types/total_balance/impls/new_dummy.rs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+impl TotalBalance {
```
Comment (politeWall) [inline src/types/total_balance/impls/new_dummy.rs:1]: TotalBalance is not define here

Comment (kenta-elys) [reply src/types/total_balance/impls/new_dummy.rs:1]: @politeWall 
Done

File: src/types/portfolio/impls/new_dummy.rs (hunk 4 above)
Comment (kenta-elys) [reply src/types/portfolio/impls/new_dummy.rs:1]: @politeWall 
Done

File: src/tests/mock/instantiate.rs (hunk 3 above)
Comment (kenta-elys) [reply src/tests/mock/instantiate.rs:9]: @politeWall 
Fixed

File: src/error.rs (hunk 2 above)
Comment (kenta-elys) [reply src/error.rs:13]: @politeWall 
Fixed

**Review by cosmic-vagabond — APPROVED** [review]

**Pull request merged**